test/unit/spec-testing.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+import {Application, CoreBindings} from '@loopback/core';
+import Dredd = require('dredd');
+import {expect} from '@loopback/testlab';
+import {HelloWorldApp} from '../..';
+
+describe('Api Spec End Points', () => {
+  let dredd;
+  before(initEnvironment);
+
+  describe('input/output test', () => {
+
+    it('passes match', done => {
+      dredd.run((err, stats) => {
+        expect(err).to.be.eql(null);
```

Comment:
This will hide any error details from Dredd (e.g. stack trace) and fail the test with an unhelpful error. Use this instead:

```ts
if (err) return done(err);
```